themes/digital.gov/layouts/guides/list.html

```diff
@@ -0,0 +1,49 @@
+{{/* Display guides in a card list that uses the guide_weight field to display each card */}}
+
+{{ define "content" }}
+
+  {{ $guidesList := slice }}
+  {{ $guidesData := slice }}
+  {{ $guidePages := slice }}
+
+  {{/* Build list of guide names from yaml files in string format */}}
+  {{ range $i, $names := $.Site.Data.guidenav }}
+    {{ $guidesList = $guidesList | append $i }}
+  {{ end }}
+
+  {{/* Iterate over list of guide names and save each as a map of strings */}}
+  {{ range $guidesList }}
+    {{ $guidesData = $guidesData | append (index $.Site.Data.guidenav .) }}
+  {{ end }}
+
+  {{/* Save each guide homepage as hugo page collection */}}
+  {{ range $guidesData }}
+    {{/* Get the first nav item from each guide, the guide homepage */}}
+    {{ range first 1 .nav }}
+      {{ $guidePages = $guidePages | append ($.Site.GetPage .path) }}
+    {{ end }}
+  {{ end }}
```

Comment:
We use the `data/guidenav` `yaml` files to populate the landing page. Hugo doesn't provide a convenient way to get the top level guide `_index.md` using the where query.